save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/Execution.kt

```diff
@@ -171,8 +173,19 @@ class Execution(
             user = null,
             execCmd = null,
             batchSizeForAnalyzer = null,
+            testSuiteSourceName = "",
         )
 
+        /**
+         * Parse and get testSuiteIds as List<Long>
+         *
+         * @param testSuiteIds
+         * @return list of TestSuite IDs
+         */
+        fun parseAndGetTestSuiteIds(testSuiteIds: String?): List<Long>? = testSuiteIds
```

Comment:
Maybe `testSuiteIdsAsList`?

Reply:
Since one of this methods is a member function, it more appropriate to use current name, take look:

```kotlin
val execution = Execution(....)
val testSuiteids = execution.parseAndGetTestSuiteIds()
```
**VS**
```kotlin
val execution = Execution(....)
val testSuiteids = execution.testSuiteIdsAsList()
```

Will leave this name as is